test/integration/element/inspec/element_spec.rb

```diff
@@ -27,3 +29,11 @@
   its('status') { should eq 200 }
   its('body') { should match '<title>Element</title>' }
 end
+
+# Branding
+
+%w(background.jpg logo.svg).each do |f|
+  describe http("127.0.0.1:8000/assets/#{f}", headers: { 'Host': 'chat.example.org' }) do
+    its('status') { should eq 200 }
```

Comment:
You might also check the size of each file to make sure it's actually what you expect.